modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -141,7 +141,7 @@ export default {
       return (window.apos.modules[this.moduleName] || {}).action;
     },
     schema() {
-      return this.moduleOptions.schema || [];
+      return (this.moduleOptions.schema || []).filter(field => apos.schema.components.fields[field.type]);
```

Comment:
Since it's done in so many places, maybe this should be done server-side in the pieces module before browserOptions are set up instead?

Reply:
okay...